src/algebra/ordered_group.lean

```diff
@@ -682,24 +682,6 @@ sub_lt_iff_lt_add'.trans (lt_add_iff_pos_left _)
 
 end ordered_add_comm_group
 
-/-
-TODO:
-The `add_lt_add_left` field of `ordered_add_comm_group` is redundant,
-and it is no longer in core so we can remove it now.
-This alternative constructor is a workaround until someone fixes this.
--/
-
-/-- Alternative constructor for ordered commutative groups,
-that avoids the field `mul_lt_mul_left`. -/
-@[to_additive "Alternative constructor for ordered commutative groups,
-that avoids the field `mul_lt_mul_left`."]
```

Comment:
This docstring was extra wrong - `ordered_add_comm_group.mul_lt_mul_left` never existed!

Reply:
Yes, the additive version of the docstring is incorrect.